lib/gruf/controllers/request.rb

```diff
@@ -87,7 +87,7 @@ def initialize(method_key:, service:, rpc_desc:, active_call:, message:)
       # @return [String] The mapped service key
       #
       def service_key
-        @service.name.to_s.underscore.tr('/', '.').gsub('.service', '')
+        @service.name.to_s.underscore.tr('/', '.').delete_suffix('.service')
```

Comment:
Hi @t0ch1k - can we add a unit test that covers your case explicitly? (that it deletes only the suffix, not all instances)